shell/shared/stats.js

```diff
@@ -137,6 +137,23 @@ if (Meteor.isServer) {
     record.computeTime = Date.now() - now;
 
     ActivityStats.insert(record);
+    if (ActivityStats.find().count() > 3) {
+      var reportSetting = Settings.findOne({_id: "reportStats"});
+      if (!reportSetting) {
+        // Setting not set yet, send out notifications and set it to false
+        globalDb.sendAdminNotification("Sandstorm.io would like you to opt-in to stats " +
```

Comment:
This sounds a bit scary. How about: "You can help Sandstorm by sending us some anonymous usage stats. Click here for more info."